package.json

```
@@ -14,7 +14,11 @@
     "load-schedule": "curl -sS https://raw.githubusercontent.com/nodejs/Release/master/schedule.json -o source/schedule.json",
     "start": "npm run serve",
     "test": "npm run test:lint && npm run test:unit && npm run test:smoke",
-    "test:lint": "standard --fix && htmllint **/*.hbs && stylint layouts/css && markdownlint \"**/*.md\" -i \"node_modules/\"",
+    "htmllint": "htmllint **/*.hbs",
```

I think a naming convention like this would be more consistent

```suggestion
    "test:lint:html": "htmllint **/*.hbs",
```

---

TBH I prefer the shorter one, but it's your call. We can keep the current scheme for scripts that run other scripts.

---

stick with the shorter one then... nbd

---

I agree with @MylesBorins ― consistency is more preferable.

---

That's just extra characters to type without any real gain. Similarly with the other individual scripts, they are not all prefixed.

I strongly suggest that you guys reconsider. Typing less is better.

---

While we're bikeshedding this, I suggest scheme `lint-xx` like we have in the Node.js repo:

https://github.com/nodejs/node/blob/e4608381683de487c3c5844875e9adb692bb8725/Makefile#L1227